src/main/java/edu/greenblitz/robotName/utils/DigitalInputMap.java

```diff
@@ -0,0 +1,41 @@
+package edu.greenblitz.robotName.utils;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+
+public class DigitalInputMap extends GBSubsystem {
```

Comment:
why do we need this? if you want to use the DIO create a `DigitalInput` at the subsystem